### DISCUSSION TITLE
1.2 Obtain configuration for MiW and KeyCloak

### DISCUSSION THREAD

**Discussion by firarasl** [eclipse-tractusx/tractusx-edc#870] [Q&A]
whats MiW and how to get these values? 

1. url: a URL where MiW is reachable
2. authorityId: this is the issuerIdentifier for MiW REST requests, please refer to the respective documentation.

from page https://github.com/eclipse-tractusx/tractusx-edc/blob/main/docs/samples/example-dataspace/README.md

**Comment by lgblaumeiser** [comment]
Closed as stale, please renew item if this is still an issue
